kaniko.go

```diff
@@ -174,6 +176,50 @@ func (p Plugin) Exec() error {
 		return fmt.Errorf("repository name to publish image must be specified")
 	}
 
+	if p.Build.LocalTarPath != "" {
+		if p.Build.Repo == "" {
+			return fmt.Errorf("destination repository name must be specified")
+		}
+
+		// Validate local tarball exists
+		if _, err := os.Stat(p.Build.LocalTarPath); os.IsNotExist(err) {
+			return fmt.Errorf("local tarball does not exist at path: %s", p.Build.LocalTarPath)
+		}
+
+		var tags []string
+		var err error
+		if p.Build.AutoTag {
+			tags, err = p.Build.AutoTags()
+			if err != nil {
+				return err
+			}
+		} else if len(p.Build.Tags) > 0 {
+			tags = p.Build.Tags
+		} else {
+			tags = []string{"latest"}
+		}
```

Comment:
Is this current behavior when no tags are specified?

Reply:
Yes, if no tags are mentioned it pushes as `latest` tag.
It's the default behaviour for kaniko/docker.